docs/organizations/billing/change-azure-subscription.md

```diff
@@ -16,59 +16,61 @@ ms.date: 07/10/2018
+   ![Select Move and then Move to another subscription](_img/change-azure-subscription/select-move-to-another-subscription.png)
 
 5. Select your target subscription and resource group.
-6. Choose **OK**.
+6. Select **OK**.
 
-## Remove billing subscription and purchase again
+## Remove the billing subscription and purchase again
 
 ### Prerequisites
 
-1. [VSTS project collection administrator or organization owner permissions](../accounts/faq-add-delete-users.md#find-owner)
-2. [The **owner** or **contributor** role on your Azure subscription](add-backup-billing-managers.md)
+* [VSTS project collection administrator or organization owner permissions](../accounts/faq-add-delete-users.md#find-owner)
+* [The owner or contributor role on your Azure subscription](add-backup-billing-managers.md)
 
 >[!NOTE]
-> When you remove the billing subscription from your organization, any paid quantities of VSTS Basic, Package Management users, Test Manager users, Microsoft-hosted Pipelines, and self-hosted Pipelines you’ve paid for this month will continue uninterrupted until the 1st of next month, but your organization will revert immediately to the Free Tier for cloud-based load testing. Removing the subscription will also cancel any non-Microsoft paid extensions without refund or credit.
+> When you remove the billing subscription from your organization, any paid quantities of VSTS Basic, Package Management users, Test Manager users, Microsoft-hosted pipelines, and self-hosted pipelines you’ve paid for this month will continue uninterrupted until the first of next month. But your organization will revert immediately to the free tier for cloud-based load testing. Removing the subscription will also cancel any non-Microsoft paid extensions without refund or credit.
 
-[Remove billing subscription](#remove-billing-subscription)
+[Remove the billing subscription](#remove-the-billing-subscription)
 
-### Remove billing subscription
+### Remove the billing subscription
 
 1. [Sign in to the Azure portal](https://portal.azure.com/) as VSTS organization owner and as Azure subscription co-administrator or greater.
 
-    If you experience browser problems with Azure,
-    make sure that you use a [supported browser](/azure/azure-preview-portal-supported-browsers-devices).
+   If you experience browser problems with Azure, make sure that you use a [supported browser](/azure/azure-preview-portal-supported-browsers-devices).
 
-2. Go to **All services** > **Team Services organizations**. 
+2. Go to **All services** > **Team Services accounts**. 
 
-   ![Choose All services and Team Services organizations](_img/change-azure-subscription/all-services-team-services-organizations.png)
+   ![Choose All services and Team Services accounts](_img/change-azure-subscription/all-services-team-services-organizations.png)
 
-3. Select your organization and remove billing.
+3. Select your organization and remove billing by selecting **Unlink**.
```

Comment:
Please omit this change, too. The portal no longer says "unlink" and master is correct

Reply:
I reverted this change.